rules/windows/defense_evasion_file_creation_mult_extension.toml

```diff
@@ -0,0 +1,70 @@
+[metadata]
+creation_date = "2021/01/19"
+maturity = "production"
+updated_date = "2021/01/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Masquerading can allow an adversary to evade defenses and better blend in with the environment. One way it occurs is
+when the name or location of a file is manipulated as a means of tricking a user into executing what they think is a
+benign file type but is actually executable code.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Executable File Creation with Multiple Extensions"
+risk_score = 47
+rule_id = "8b2b3a62-a598-4293-bc14-3d5fa22bb98f"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+file where event.type == "creation" and file.extension : "exe" and
+  file.name:(
```

Comment:
maybe u can add also same for macOS commonly abused extensions (.app and .pkg and .dmg): *.doc.app, *.pdf.app ... *.doc.pkg *.pdf.pkg *.doc.dmg